### DISCUSSION TITLE
How to initialize Valtio

### DISCUSSION THREAD

**Discussion by itsGKRam** [pmndrs/valtio#303] [Q&A]
Cannot access '__WEBPACK_DEFAULT_EXPORT__' before initialization

This is the error I am getting in NECT.js

![image](https://user-images.githubusercontent.com/85800297/147744779-67e45921-31e5-4402-b1ed-9e3f4a9666f6.png)

**Comment by dai-shi** [comment]
I wonder if anyone has a working example in next.js.

**Reply by itsGKRam** [reply]
> I wonder if anyone has a working example in next.js.

@dai-shi 
can you help me out I am stuck at the middle of my project

**Reply by dai-shi** [reply]
Can you create a **minimal** reproduction with codesandbox or stackblitz?

**Reply by dai-shi** [reply]
Does any of sandboxes help? https://codesandbox.io/search?refinementList%5Btemplate%5D%5B0%5D=next&refinementList%5Bnpm_dependencies.dependency%5D%5B0%5D=valtio&page=1&configure%5BhitsPerPage%5D=12
